src/uikit.cs

```diff
@@ -700,6 +700,18 @@ public partial interface NSTextStorage {
 		[Since (7,0)]
 		[Notification, Internal, Field ("NSTextStorageDidProcessEditingNotification")]
 		NSString DidProcessEditingNotification { get; }
+
+		// we need to hide this methods inherited from the NSMuttableAttributedString since the class does not implement them
+#if XAMCORE_4_0
+		[New, Abstract]
```

Comment:
This will introduce a new member, which means that it won't be called when the instance is the base type:

    Console.WriteLine (((NSMutableAttributedString) obj).LowLevelValue);

will still not work.

Instead I think you should override and make abstract (in manual code):

    public IntPtr override abstract LowLevelValue { get; }

It looks weird to have both `override` and `abstract`, but the C# compiler accepts it (I just checked).

However I'm not sure if the `[Export]` attribute will be required on the abstract member or not (a test would be nice, but I guess that's difficult until XAMCORE_4_0 is defined).